web/src/routes/(user)/people/+page.svelte

```diff
@@ -223,6 +223,29 @@
     }
   };
 
+  const handleToggleFavorite = async (detail: PersonResponseDto) => {
+    try {
+      const updatedPerson = await updatePerson({
+        id: detail.id,
+        personUpdateDto: { isFavorite: !detail.isFavorite },
+      });
+
+      people = people.map((person: PersonResponseDto) => {
+        if (person.id === updatedPerson.id) {
+          return updatedPerson;
+        }
+        return person;
```

Comment:
When can you possibly not find your person in here?

Reply:
Not exactly sure what you mean, this is inside of the map, its replacing the updatedPerson inside of the person list.

And I just copied the code for the other handlers in that file :D

Reply:
What I had in mind is that `const index = findIndex((person) => person.id === detail.id)` followed by `people[index] = updatedPerson` should also work and is imo more intuitive to read. (plus it should be a touch more efficient lol)

Reply:
Ahh right, I refactored it like that now 👍